### PR TITLE
Rework the Skin Compendium

### PR DESCRIPTION
**Pull request by RockRoller01**
The compendium on the forum got split into categories, applied the same to the wiki skin compendium.

### CODE REVIEW

File: wiki/Skinning/Compendium/en.md [hunk 1]
```diff
@@ -1,373 +1,10 @@
...
-- [東方Project - Hinanawi Tenshi](/community/forums/topics/211868) by [Elzapatoverde](/users/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
-- [東方Project - Kirisame Marisa](/community/forums/topics/530165) by [Elzapatoverde](/users/3717733)![][o!s]![][o!c]
+Compendium of all completed osu! skins, splitted into the following categories:
+[Anime](/wiki/Skinning/Compendium/Anime)
+[Eyecandy](/wiki/Skinning/Compendium/Eyecandy)
+[Minimalsitic & Flat](/wiki/Skinning/Compendium/Minimalsitic_&_Flat)
+[Touhou](/wiki/Skinning/Compendium/Touhou)
+[Video Games](/wiki/Skinning/Compendium/Video_Games)
+[Vocaloid](/wiki/Skinning/Compendium/Vocaloid)
+[Others](/wiki/Skinning/Compendium/Others)
```
Comment (MegaApplePi) [inline wiki/Skinning/Compendium/en.md]: change lines 4 - 10 to:
```md
-    [Anime](/wiki/Skinning/Compendium/Anime)
-    [Eyecandy](/wiki/Skinning/Compendium/Eyecandy)
-    [Minimalistic & Flat](/wiki/Skinning/Compendium/Minimalsitic_&_Flat)
-    [Touhou](/wiki/Skinning/Compendium/Touhou)
-    [Video Games](/wiki/Skinning/Compendium/Video_Games)
-    [Vocaloid](/wiki/Skinning/Compendium/Vocaloid)
-    [Others](/wiki/Skinning/Compendium/Others)
```

File: wiki/Skinning/Compendium/Anime/en.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+Compendium of all completed anime osu! skins.
+
+- ![][o!s]  - means the skin contains osu!standard elements.
+- ![][o!c]  - means the skin contains osu!catch elements.
+- ![][o!t]  - means the skin contains osu!taiko elements.
+- ![][o!m]  - means the skin contains osu!mania elements.
+
+## A
+
+- [Aiko](/community/forums/topics/764430?start=6704425) by [ShovelKun](/users/9948284)![][o!s]
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/Anime/en.md]: is the `?start` part needed?

File: wiki/Skinning/Compendium/Anime/en.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+- [Elek-Taiko](/community/forums/topics/357445) by [Elekyon](/users/4460512)![][o!t]
+- [Elf](/community/forums/topics/631673) by [atturbo555](/users/5730417)![][o!s]
+- [Ene Edition](/community/forums/topics/434972) by [atturbo555](/users/5730417)![][o!s]![][o!t]![][o!c]![][o!m]
+- [ENE06](/community/forums/topics/473369) by [Choilicious](/users/2129634)![][o!s]![][o!c]
+
+## F
+
+- [Fall in Love x 4 Tune](/community/forums/topics/445514) by [atturbo555](/users/5730417)![][o!s]![][o!m]
+- [Feu Sacré](/forum/p/6659659/) by [My Angel Denis](/users/8984574)![][o!s]![][o!m]
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/Anime/en.md]: change to /community/forums/topics/755152/

File: wiki/Skinning/Compendium/Touhou/en.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Touhou
+Compendium of all completed touhou osu! skins.
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/Touhou/en.md]: Compendium of all completed osu! skins that are based on Touhou Project.

File: wiki/Skinning/Compendium/Video_Games/en.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Video Games
+Compendium of all completed osu! skins that are based on other games/game characters.
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/Video_Games/en.md]: / -> or

File: wiki/Skinning/Compendium/Vocaloid/en.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Vocaloid
+Compendium of all completed vocaloid osu! skins.
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/Vocaloid/en.md]: Compendium of all completed osu! skins that are based on Vocaloid characters.

File: wiki/Skinning/Compendium/en.md [hunk 7]
```diff
@@ -1,373 +1,10 @@
...
-- [[Nightcore HD]](/community/forums/topics/135588) by [evil_genius](/users/2526988)![][o!s]![][o!t]![][o!c]![][o!m]
-- [»multiberry«](/community/forums/topics/607808) by [atturbo555](/users/5730417)![][o!s]
-- [Ōkami - 大](/community/forums/topics/275293) by [Elzapatoverde](/users/3717733)![][o!s]
-- [▰ shocked! ▰](/community/forums/topics/248517) by [Angelo](/users/2094431)![][o!s]
-- [♥ loveHeart](/community/forums/topics/359459) by [Rebecca](/users/6509768)![][o!s]
-- [✿ 空母ヲ級 (Aircraft carrier Wo-Class)](/community/forums/topics/478380) by [Choilicious](/users/2129634)![][o!s]
-- [東方Project - Hata no Kokoro](/community/forums/topics/224323) by [Elzapatoverde](/users/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
-- [東方Project - Hinanawi Tenshi](/community/forums/topics/211868) by [Elzapatoverde](/users/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
-- [東方Project - Kirisame Marisa](/community/forums/topics/530165) by [Elzapatoverde](/users/3717733)![][o!s]![][o!c]
+Compendium of all completed osu! skins, splitted into the following categories:
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/en.md]: Compendium of all completed osu! skins. They are currently splitted into the following categories:

File: wiki/Skinning/Compendium/Anime/en.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Anime
+Compendium of all completed anime osu! skins.
```
Comment (Joehuu) [inline wiki/Skinning/Compendium/Anime/en.md]: Add a line between header and description for all.

File: wiki/Skinning/Compendium/Anime/en.md (hunk 2 above)
Comment (RockRoller01) [reply wiki/Skinning/Compendium/Anime/en.md]: Wdym?

File: wiki/Skinning/Compendium/Anime/en.md (hunk 8 above)
Comment (RockRoller01) [reply wiki/Skinning/Compendium/Anime/en.md]: Is that something that is needed or just something thats suggested?

**Pull request closed without merging**